packages/events/src/FederatedEventMap.ts

```diff
@@ -16,6 +16,7 @@ export type FederatedEventMap = {
     pointertap: FederatedPointerEvent;
     pointerup: FederatedPointerEvent;
     pointerupoutside: FederatedPointerEvent;
+    pointercancel: FederatedPointerEvent;
```

Comment:
Nice catch! And I think it is better to move this line to between `mouseupoutside` and `pointerdown` (so that the map is in alphabetical order).

By the way, I checked [EventBoundary.ts](https://github.com/pixijs/pixijs/blob/3ba43e50030641e2cb028c7c265e80ce0fdf6fff/packages/events/src/EventBoundary.ts) and found that the `mouseenter`, `mouseleave`, `pointerenter`, `pointerleave` are also missing. Can you add them?

Reply:
added!
I'm not sure the consequences - but just making sure,
the mouseenter, mousedown etc. events are supposed to be FederatedPointerEvents and not FederatedMouseEvents?

Reply:
Seems that `FederatedMouseEvents` is only used as the base class of `FederatedPointerEvents` and `FederatedWheelEvent`, so I think it is OK to use `FederatedPointerEvents`.